symfit/core/fit_results.py

```diff
@@ -57,13 +73,13 @@ def __str__(self):
             res += '{:10}{} {}\n'.format(p.name, value_str, stdev_str, width=20)
 
         res += 'Fitting status message: {}\n'.format(self.status_message)
-        res += 'Number of iterations:   {}\n'.format(self.infodict['nfev'])
+        res += 'Number of iterations:   {}\n'.format(self.iterations)
         res += 'Objective:              {}\n'.format(self.objective)
         res += 'Minimizer               {}\n'.format(self.minimizer)
-        try:
-            res += 'Regression Coefficient: {}\n'.format(self.r_squared)
-        except AttributeError:
-            pass
+
+        res += '\nGoodness of fit qualifiers:\n'
+        res += '\n'.join('{:20} {}'.format(gof, value)
+                         for gof, value in sorted(self.gof_qualifiers.items()))
```

Comment:
You can make the `:20` dependent on the largest length of any GoF qualifier ;) See the comment above.